packages/opencensus-exporter-stackdriver/src/stackdriver-cloudtrace.ts

```diff
@@ -49,6 +48,7 @@ export class StackdriverTraceExporter implements Exporter {
   logger: Logger;
   failBuffer: SpanContext[] = [];
   private RESOURCE_LABELS: Promise<Record<string, AttributeValue>>;
+  private _cloudTrace: cloudtrace_v2.Cloudtrace;
```

Comment:
nit the underscore is not needed since TS has private visibility

Reply:
done